pymeasure/instruments/anritsu/anritsuMS464xB.py

```diff
@@ -0,0 +1,912 @@
+        value   description
+        =====   ==================================================
+        AUTO    Automatic triggering
+        MAN     Manual triggering
+        EXTT    Triggering from rear panel BNC via the GPIB parser
+        EXT     External triggering port
+        REM     Remote triggering
+        =====   ==================================================
+        """,
+        values=["AUTO", "MAN", "EXTT", "EXT", "REM"],
+        validator=strict_discrete_set,
+    )
+
+    external_trigger_type = Instrument.control(
+        ":TRIG:EXT:TYP?", ":TRIG:EXT:TYP %s",
+        """Control the type of trigger that will be associated with the external trigger.
+
+        Valid values are POIN (for point), SWE (for sweep), CHAN (for channel), and ALL.
+        """,
+        values=TRIGGER_TYPES,
+        validator=strict_discrete_set,
+    )
+
+    external_trigger_delay = Instrument.control(
+        ":TRIG:EXT:DEL?", ":TRIG:EXT:DEL %g",
+        """Control the the delay time of the external trigger.
+
+        Valid values are between 0 [s] and 10 [s] in steps of 1e-9 [s] (i.e. 1 ns).
+        """,
+        values=[0, 10],
+        validator=strict_range,
+    )
+
+    external_trigger_edge = Instrument.control(
+        ":TRIG:EXT:EDG?", ":TRIG:EXT:EDG %s",
+        """Control the which edge is used for triggering of the external trigger.
+
+        Valid values are POS (for positive or leading edge) or NEG (for negative or trailing edge).
+        """,
+        values=["POS", "NEG"],
+        validator=strict_discrete_set,
+    )
+
+    external_trigger_handshake = Instrument.control(
+        ":TRIG:EXT:HAND?", ":TRIG:EXT:HAND %s",
+        """Control status of the external trigger handshake. """,
+        values={True: 1, False: 0},
+        map_values=True,
+    )
+
+    remote_trigger_type = Instrument.control(
+        ":TRIG:REM:TYP?", ":TRIG:REM:TYP %s",
+        """Control the type of trigger that will be associated with the remote trigger.
+
+        Valid values are POIN (for point), SWE (for sweep), CHAN (for channel), and ALL.
+        """,
+        values=TRIGGER_TYPES,
+        validator=strict_discrete_set,
+    )
+
+    manual_trigger_type = Instrument.control(
+        ":TRIG:MAN:TYP?", ":TRIG:MAN:TYP %s",
+        """Control the type of trigger that will be associated with the manual trigger.
+
+        Valid values are POIN (for point), SWE (for sweep), CHAN (for channel), and ALL.
+        """,
+        values=TRIGGER_TYPES,
+        validator=strict_discrete_set,
+    )
+
+    def trigger(self):
+        """ Triggers a continuous sweep from the remote interface. """
+        self.write("*TRG")
+
+    def trigger_single(self):
+        """ Triggers a single sweep with synchronization from the remote interface. """
+        self.write(":TRIG:SING")
+
+    def trigger_continuous(self):
+        """ Triggers a continuous sweep from the remote interface. """
```

Comment:
In imperative mode 😁 :
```suggestion
        """ Trigger a continuous sweep from the remote interface. """
        self.write("*TRG")

    def trigger_single(self):
        """ Trigger a single sweep with synchronization from the remote interface. """
        self.write(":TRIG:SING")

    def trigger_continuous(self):
        """ Trigger a continuous sweep from the remote interface. """
```